### PR TITLE
Add .wsgi extension

### PR DESCRIPTION
**Pull request by asottile**
For python wsgi applications it's relatively common to list your entrypoint in a `.wsgi` file

### CODE REVIEW

File: identify/extensions.py [hunk 1]
```diff
@@ -117,6 +117,7 @@
     'whl': {'binary', 'wheel', 'zip'},
     'woff': {'binary', 'woff'},
     'woff2': {'binary', 'woff2'},
+    'wsgi', {'text', 'wsgi', 'python'},
```
Comment (chriskuehl) [inline identify/extensions.py]: derp, nice comma

Comment (asottile) [reply identify/extensions.py]: lol gdi

Comment (asottile) [reply identify/extensions.py]: that's what I get by contributing from github ui and not using `pre-commit` :'(

**Comment by asottile** [comment]
I split out #29 a while back on designing some sort of alias system -- haven't really put much thought into it yet though!

**Pull request merged**